README.rst

```diff
@@ -55,7 +55,7 @@ For more help on using Mesa, check out the following resources:
 * `Email list`_
 * `PyPI`_
 
-.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/intro-tutorial.html
+.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/tutorials/intro_tutorial.html
```

Comment:
@snargleplax This is the one that I am looking at as the wrong link. The domain part is the part that needs to be fixed as well. Am I looking at this wrong?